__tests__/parser.ts

```diff
@@ -107,7 +108,7 @@ it("can group fingerprintable window objects", async () => {
   expect(output["NAVIGATOR"]).toEqual(NAVIGATOR_SYMBOLS);
   expect(output["SCREEN"]).toEqual(SCREEN_SYMBOLS);
   expect(output["MEDIA_DEVICES"]).toEqual(MEDIA_DEVICES_SYMBOLS);
-});
+}, 60000);
```

Comment:
What's the 60000 here doing? Also the two above.

Reply:
It changes the time limit for a test to run. The default is 5,000 but sometimes it goes over that causing the test case to fail.

Reply:
milliseconds? @samatt do you have thoughts about a good default here?

Is this something new in the upgrade, or was it like that before?

Reply:
Oh never mind this is just in __tests__. Never mind.